s3/replication/manager/tests/system/load_transfer_test.py

```diff
@@ -261,49 +261,71 @@ async def run_load_test():
     polling_count = test_config.polling_count
 
     while jobs_running and polling_count != 0:
-        job_status_task_list = []
-        for job_id in posted_jobs_set:
-            logger.debug(
-                "Checking status for job: job_id= {}".format(job_id))
-
-            job_status_task = asyncio.ensure_future(manager_session.get(
-                url + '/jobs/' + job_id))
-            job_status_task_list.append(job_status_task)
-
-        # jobs status contains list of response from GET /jobs/<job_id> for
-        # each
-        logger.debug("Waiting for all GET {}/jobs/<job_id> response...".
-                     format(url))
-        get_job_response_list = await asyncio.gather(*job_status_task_list)
-        # logger.debug("get_job_response_list: {}".format(
-        #     get_job_response_list))
-
-        for get_job_resp in get_job_response_list:
-            job_status = await get_job_resp.json()
-            job_id = job_status["job"]["job_id"]
-            job_state = job_status["job"]["state"]
-            logger.debug(
-                "job_state = {} for job_id {}".format(
-                    job_state, job_id))
-            if job_state == "COMPLETED":
-                # remove completed job from polling list
-                posted_jobs_set.remove(job_id)
+        async with manager_session.get(url + '/jobs?count=completed') as response:
+            logger.info(
+                'GET jobs returned http Status: {}'.format(response.status))
+            jobs = await response.json()
 
-        if len(posted_jobs_set) == 0:
+        completed_count = jobs['count']
+        logger.info("completed jobs count : {}".format(completed_count))
+
+        if completed_count == test_config.count_of_obj:
             # No jobs pending.
             jobs_running = False
             logger.info("All jobs completed.")
         else:
             # There are atleast some running jobs, give time to complete.
+            print("CCOUNT OF OBJES ARE : {}".format(test_config.count_of_obj))
             logger.debug(
                 "Pending status for total {} jobs.".format(
-                    len(posted_jobs_set)))
+                    test_config.count_of_obj - completed_count))
             logger.info("Waiting for {} secs before polling job status...".
                         format(test_config.polling_wait_time))
             time.sleep(test_config.polling_wait_time)
 
         polling_count -= 1
 
+    inprogress_count = 0
+    queued_count = 0
+    async with manager_session.get(url + '/jobs?count=inprogress') as response:
+        logger.info(
+            'GET jobs returned http Status: {}'.format(response.status))
+        jobs = await response.json()
+
+        inprogress_count = jobs['count']
+        logger.info("INPROGRESS COUNT is : {}".format(inprogress_count))
```

Comment:
`s/INPROGRESS COUNT/In-progress jobs count`

Similar case change at messages below.

Reply:
done